### ISSUE TITLE
Use incoherent approximation in the epithermal energy region

### ISSUE THREAD

**Issue by marquezj** [mctools/ncrystal#72]
Coherent scattering at high incident energies can be approximated by incoherent scattering, by means of replacing the sum over reciprocal space points with an integral (see, for instance V.F. Turchin, "Slow Neutrons", page 60)

This could be used to slightly improve the accuracy of the coherent scattering, by extending the sum with an integral (instead of extrapolating the last Bragg edge as 1/E), or to accelerate the code by taking into consideration a smaller number of planes without compromising the total scattering cross section:
<div align="center">
<a href="https://www.codecogs.com/eqnedit.php?latex=\frac{\lambda^2}{2V_{uc}}&space;\sum_{hkl}^{\lambda&space;\leq&space;2d_{hkl}}&space;d_{hkl}&space;\left|F\left(\vec{\tau}_{hkl}&space;\right&space;)\right|^2&space;\approx&space;\frac{\lambda^2}{2V_{uc}}&space;\sum_{hkl}^{\lambda&space;\leq&space;2d_{hkl}^\text{cutoff}&space;}&space;d_{hkl}^\text{cutoff}&space;\left|F\left(\vec{\tau}_{hkl}&space;\right&space;)\right|^2&plus;\int_{}\left(\text{Incoherent&space;model}&space;\right&space;)" target="_blank"><img src="https://latex.codecogs.com/gif.latex?\frac{\lambda^2}{2V_{uc}}&space;\sum_{hkl}^{\lambda&space;\leq&space;2d_{hkl}}&space;d_{hkl}&space;\left|F\left(\vec{\tau}_{hkl}&space;\right&space;)\right|^2&space;\approx&space;\frac{\lambda^2}{2V_{uc}}&space;\sum_{hkl}^{\lambda&space;\leq&space;2d_{hkl}^\text{cutoff}&space;}&space;d_{hkl}^\text{cutoff}&space;\left|F\left(\vec{\tau}_{hkl}&space;\right&space;)\right|^2&plus;\int_{}\left(\text{Incoherent&space;model}&space;\right&space;)" title="\frac{\lambda^2}{2V_{uc}} \sum_{hkl}^{\lambda \leq 2d_{hkl}} d_{hkl} \left|F\left(\vec{\tau}_{hkl} \right )\right|^2 \approx \frac{\lambda^2}{2V_{uc}} \sum_{hkl}^{\lambda \leq 2d_{hkl}^\text{cutoff} } d_{hkl}^\text{cutoff} \left|F\left(\vec{\tau}_{hkl} \right )\right|^2+\int_{}\left(\text{Incoherent model} \right )" /></a>
</div>

**Comment by marquezj** [comment]
I am trying to remember what we were discussing here, because the equation is just a concept: using the incoherent approximation for Bragg scattering in the epithermal limit, to reduce the artifacts when `dcutoff` is reduced:

<img width="668" height="59" alt="Image" src="https://github.com/user-attachments/assets/e8bcc334-fae0-4f8e-bc06-63f1923d55ee" />

It might be something along these lines (this is from "Low Energy Neutron Physics" by Gurevich and Tarasov):

![Image](https://github.com/user-attachments/assets/35fb1744-ebcf-4df0-8047-87f9eb188862)

**Comment by tkittel** [comment]
Yeah ok, but that doesn't tell us how to estimate the contribution from any planes left out due to a dspacing cut-off, does it?
